ui/src/sandbox.rs

```diff
@@ -354,13 +357,9 @@ impl Sandbox {
         cmd
     }
 
-    fn miri_command(&self, edition: Option<Edition>) -> Command {
+    fn miri_command(&self, req: impl EditionRequest) -> Command {
         let mut cmd = self.docker_command(None);
-        if let Some(edition) = edition {
-            // set the env var that will cause modify-cargo-toml inside the docker
-            // container to set this in the `Cargo.toml`.
```

Comment:
I think having some comment about how these pieces work together somewhere is really helpful, it took me a bit to figure this out when I wrote the patch for Miri.^^  The propagation of information through 3-5 layers here is far from obvious.